stubs/tree-sitter-languages/METADATA.toml

```diff
@@ -0,0 +1,5 @@
+version = "1.2.*"
```

Comment:
```suggestion
version = "1.3.*"
```

It was released two days ago ;)

Reply:
Indeed. That release was because I made a couple pull requests there :)